webdriver/support/fixtures.py

```diff
@@ -115,3 +117,83 @@ def inner(path, query="", fragment=""):
                                   fragment))
         return rv
     return inner
+
+def switch_to_inactive(session):
+    """Create a new browsing context, switch to that new context, and then
+    delete the context."""
+    def switch_to_inactive():
+        initial = session.send_session_command("GET", "window/handles")
+
+        session.send_session_command("POST",
+                                     "execute/sync",
+                                     dict(script="window.open();", args=[]))
+        with_new = session.send_session_command("GET", "window/handles")
+
+        assert len(initial) == len(with_new) - 1
+
+        new_handle = (set(with_new) - set(initial)).pop()
```

Comment:
I think that is accounted for by casting the Lists into sets. Although this code is going away, it's the same pattern used in the previously-existing `create_window` fixture function, so please let me know if that's faulty.